shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -68,6 +86,17 @@ public class PlatformViewsController implements PlatformViewsAccessibilityDelega
   // dispatched.
   private final AccessibilityEventsDelegate accessibilityEventsDelegate;
 
+  // TODO(mattcarroll): Refactor overall platform views to facilitate testing and then make
+  // this private. This is visible as a hack to facilitate testing. This was deemed the least
+  // bad option at the time of writing.
```

Comment:
Is this TODO still relevant?

If so we should have a bug filed explaining the context. Given the time and nature of this reland I suspect it's not relevant.

Reply:
I took this from the previous code, could we do the clean up in a separate PR?

Reply:
I'd be fine with just deleting the TODO here, but don't consider this blocking. Filed https://github.com/flutter/flutter/issues/106443 to track.